terraform/aws/analytical-platform-data-production/lakeformation-external-data/digital-prisons-reporting-preproduction/main.tf

```diff
@@ -1,13 +1,13 @@
-# Module: lake_formation_analytical_platform_data_prod
-module "lake_formation_analytical_platform_data_prod" {
-  source = "github.com/ministryofjustice/terraform-aws-analytical-platform-lakeformation?ref=6fab8677e457c2e276fa1feec8ee83bbccc1220a"
+# # Module: lake_formation_analytical_platform_data_prod
```

Comment:
Are you likely to be bringing this back in future? If not, can you delete rather than comment out this code?

Reply:
Ah I wasn't aware of that - I will defer to JW's greater knowledge of terraform! Happy for you to remove the commented out code in a separate PR